helm/hpcc/templates/network.yaml

```diff
@@ -157,6 +157,25 @@ spec:
         matchLabels:
           app: eclservices
 ---
+{{ range .Values.esp -}}
+{{- if has .name $.Values.global.visibilities.exposedServices }}
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: {{ .name }}
+spec:
+  podSelector:
+    matchLabels:
+      app: {{ .name }}
+  ingress:
+{{- if .service.ingress }}
+{{ toYaml .service.ingress | indent 2}}
```

Comment:
is it correct for there to be an ingress section if the component defines visibility: local ?

Reply:
I think it doesn't matter the visibility local or global. The networkpolicies for ingress are needed for exposed services.